app/views/tags/show.html.erb

```diff
@@ -23,6 +23,9 @@
   <!--main content-->
   <p><%= ts("This tag belongs to the %{tag_class} Category.", :tag_class => @tag.class::NAME) %>
   <% if @tag.canonical %>
+    <% if @tag.is_a?(Fandom) %>
+      A list of all the Relationship tags in this <%= link_to ts('fandom is available'), fandom_path(@tag.name) %>.
```

Comment:
Should this be a `<%= ts() %>`?

Also, I'm afraid this link text won't quite work -- an accessible link describes the designate page even when taken out of context, and "fandom is available" doesn't mean much if you just say it to someone who's not looking at the whole sentence. It would be better if the words "Relationship tags in this fandom" were the link.